libs/vre/core/error-handler/src/lib/js-lib-parsed-error.ts

```diff
@@ -0,0 +1,5 @@
+export class JsLibParsedError extends Error {
```

Comment:
Typo? Present tense form sounds more natural.
```suggestion
export class JsLibParseError extends Error {
```